Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_HomeIndexView.1.txt

```diff
@@ -9,9 +9,9 @@
     <title>Swift Package Index</title>
     <meta name="twitter:title" content="Swift Package Index"/>
     <meta name="og:title" content="Swift Package Index"/>
-    <meta name="description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
-    <meta name="twitter:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
-    <meta name="og:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
+    <meta name="description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,544 packages and 38,410 versions."/>
+    <meta name="twitter:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,544 packages and 38,410 versions."/>
+    <meta name="og:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,544 packages and 38,410 versions."/>
```

Comment:
Number changed because now they are taken from the mock and not hard-coded anymore.